src/lsp/cobol_parser/text_tokenizer.ml

```diff
@@ -380,13 +380,14 @@ let tokens_of_word { persist = { lexer; _ }; _ }
   : text_word with_loc -> tokens * Parser_diagnostics.t =
   fun { payload = c; loc } ->
   let tok t = [t &@ loc], Parser_diagnostics.none in
-  let alphanum ~hexadecimal str qte =
+  let alphanum ~hexadecimal ~repr str qte =
```

Comment:
Considering this is quite local, you should just `?(repr = Native_bytes)` instead.

Reply:
Maybe. I don't like optional arguments much though (it's easy to forget about them).